resources/views/community/components/user/recent-progress.blade.php

```diff
@@ -0,0 +1,85 @@
+@props([
+    'username' => '',
+    'userScoreData' => [],
+])
+
+<?php
+use Illuminate\Support\Carbon;
+?>
+
```

Comment:
Now might be a good time to inject a "missing data" statement if there is no recent score data.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/cd37f176-786b-41c9-b944-4289dc30860c)

instead of 

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9c13b13d-11b6-45b8-8a52-0015ece21f6b)

Reply:
Good idea, this also keeps us from unnecessarily loading all that JS.

I've gone with "No points earned in the last 14 days." as the label.